fastparquet/util.py

```diff
@@ -1,22 +1,23 @@
+from collections import defaultdict
 import copy
+from distutils.version import LooseVersion
+from functools import lru_cache
 import io
 import struct
-import numpy as np
 import os
-import os.path
 import operator
-import pandas as pd
 import re
 import numbers
-from collections import defaultdict
-from distutils.version import LooseVersion
-from functools import lru_cache
-import pandas
 
+import numpy as np
+import pandas as pd
 from pandas.api.types import is_categorical_dtype
 
-PANDAS_VERSION = LooseVersion(pandas.__version__)
-created_by = "fastparquet-python version 1.0.0 (build 111)"
+from fastparquet import __version__
+
+PANDAS_VERSION = LooseVersion(pd.__version__)
+created_by = (f"fastparquet-python version {__version__.rsplit('.',1)[0]} "
```

Comment:
So I think this should be 
f"fastparquet-python version {__version__} (build 0)"

(i.e., the build part contains *no information*, because we do not make incremental builds, only new versions.

I hope this has not been too annoying for you!

Reply:
This should be ok now so.
This is fine @martindurant I am learning new stuff :)
Thanks to bear with me, I can be slow to understand. Thanks for your patience and your help!